docs/sources/flow/reference/components/module.http.md

```diff
@@ -0,0 +1,165 @@
+---
+title: module.http
+labels:
+  stage: beta
+---
+
+# module.http
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`module.http` is a *module loader* component. A module loader is a Grafana Agent Flow
+component which retrieves a [module][] and runs the components defined inside of it.
+
+`module.http` embeds a [remote.http][] component to retrieve the module from a remote
+HTTP server. This allows for the use of a single module loader, rather than a `remote.http`
+component paired with a [module.string][] component.
```

Comment:
```suggestion
`module.http` embeds a [remote.http][] component to retrieve the module from a remote
HTTP server. This module allows you to use a single module loader, rather than a `remote.http`
component paired with a [module.string][] component.
```